figures/scripts/fig4-tp53-telomerase-panels.R

```diff
@@ -414,4 +415,88 @@ cowplot::ggdraw(legend)
 dev.off()
 
 
+## Survival analysis forest plot -----------------------------------------------------------------
+
+survival_dir <- file.path(analyses_dir, "survival-analysis", "results")
+survival_result <- readRDS(file.path(
+  survival_dir,
+  "cox_additive_terms_tp53_telomerase_hgg.RDS"
+))
+
+# Convert survival model result to data frame, and exponentiate estimates/CIs to get HRs
+survival_df <- broom::tidy(survival_result) %>%
+  mutate(estimate = exp(estimate),
+         conf.low = exp(conf.low),
+         conf.high = exp(conf.high))
+
+# Forest plot of the model
+forest_plot <- ggplot(survival_df) +
+  aes(x = estimate, 
+      y = term,
+      fill = term
+  ) + 
+  # add CI first so line doesn't cover open point
+  geom_errorbarh(
+    aes(
+      xmin = conf.low,
+      xmax = conf.high,
+    ),
+    height = 0.15,
+    size = 0.65
+  ) + 
+  geom_point(
+    size = 3,
+    shape = 23
+  ) +
+  # Point fill based on sigificance
+  scale_fill_manual(
+    values = c("black", "black", "white"),
+    guide = FALSE # turn off legend
+  ) + 
+  # Vertical guiding line at 1
+  geom_vline(
+    xintercept = 1, 
+    linetype = 3
+  ) +
+  labs(
+    x = "Hazard Ratio ± 95% CI",
+    y = "Coefficient"
+  ) + 
+  # Update term labels for pub-ready
+  scale_y_discrete(
+    labels = c(
+      "HGG group (HGG)",
+      "Telomerase score",
+      "TP53 score"
```

Comment:
```suggestion
    labels = c(
      "HGG group: HGG",
      "Telomerase score",
      "TP53 score"
```
I think this will help in showing this refers to HGG as the group shown on the plot, since it seemed to be convention.